tests/test_vectors.py

```diff
@@ -96,7 +96,6 @@ def _open(file):
     # TEXT:
     _good_file(u'subfieldVarUInt.ion'),  # TODO amzn/ion-python#34
     _good_file(u'subfieldVarUInt32bit.ion'),  # TODO amzn/ion-python#34
-    _equivs_file(u'timestampsLargeFractionalPrecision.ion'),  # TODO amzn/ion-python#35
```

Comment:
Why did this get removed?

Reply:
Removal was outlined in the original issue: https://github.com/amzn/ion-python/issues/35
It's so we can tests large fractional precisions now.

Reply:
Oh is this just the skip list?  Okay, that makes sense then.